### PR TITLE
Add org.freedesktop.LinuxAudio.Plugins.NeuralAmpModeler

### PR DESCRIPTION
**Pull request by runiq**
Resubmission of https://github.com/flathub/flathub/pull/4271.

### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/mikeoliphant/neural-amp-modeler-lv2/issues/33
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
  - I am not sure what to make of this requirement in the context of a Linux audio plugin such as this. I used the [template employed by other Linux audio plugins](https://github.com/flathub/org.freedesktop.LinuxAudio.BaseExtension). Is there something else I need to do here?
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*
  - The `nam-fixes.patch` file removes a temporary workaround that has been introduced for GCC versions < 9.3.0 (IIRC). The current FDO runtime ([22.08](https://github.com/flathub/org.freedesktop.LinuxAudio.BaseExtension/blob/ed3ef4db629eb34165eedb6240fbcfd986ca1333/org.freedesktop.LinuxAudio.BaseExtension.json#L4-L5) at the time of PR submission) uses [GCC 12.2](https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/blob/freedesktop-sdk-22.08.12.1/elements/bootstrap/include/gcc-source.yml?ref_type=tags#L2-4) which does not need that workaround anymore (in fact, leaving the workaround in place results in an error, aborting the build).

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
apparently some submodules aren't available. I'll have a look at it.

File: org.freedesktop.LinuxAudio.Plugins.NeuralAmpModeler.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      - "install -Dm644 deps/NeuralAmpModelerCore/LICENSE ${FLATPAK_DEST}/share/licenses/NeuralAmpModeler/LICENSE"
+      - "install -Dm644 --target-directory=${FLATPAK_DEST}/share/metainfo org.freedesktop.LinuxAudio.Plugins.NeuralAmpModeler.metainfo.xml"
+      - "appstream-compose --basename=${FLATPAK_ID} --prefix=${FLATPAK_DEST} --origin=flatpak ${FLATPAK_ID}"
+    cleanup:
+      - lib
+    sources:
+      - type: git
+        url: https://github.com/mikeoliphant/neural-amp-modeler-lv2
+        tag: 0.1.0
+        commit: 0aad7c32b6806c826848b21bb1d2699eb79de29b
```
Comment (hfiguiere) [inline org.freedesktop.LinuxAudio.Plugins.NeuralAmpModeler.yml:26]:
```suggestion
        tag: 0.1.1
        commit: c990eedf332fe0059042015f6466ab356a2d5fee
```

**Comment by hfiguiere** [comment]
bot, build org.freedesktop.LinuxAudio.Plugins.NeuralAmpModeler

**Comment by runiq** [comment]
Hi, sorry for the long hiatus, needed to take some time off...

I have a question: Should I use the 22.08 runtime, or should I switch to 23.08 instead?

**Comment by hfiguiere** [comment]
Note that 23.08 may bring more JUCE or VST3 sdk patches...

**Comment by hfiguiere** [comment]
bot, build org.freedesktop.LinuxAudio.Plugins.NeuralAmpModeler

**Comment by runiq** [comment]
Okay, appears to work. I'm using [this dry track for testing](https://www.dropbox.com/s/8h1j30lz6mhfm84/Dry%20Guitar.wav?dl=0) and [this model](https://tonehunt.org/helgab/23bc8c03-f731-49e1-9c29-5acf7bd15a13), mostly because it comes with a video to compare. 

I imported the WAV track into an Ardour track, added the NAM plugin with the downloaded model, and pressed Play.

Sounds… good to me? Though I don't have the original to compare.

**Comment by hfiguiere** [comment]
it sounded fine this time.

**Pull request closed without merging**